src/main/java/com/ibm/cloudant/kafka/connect/CloudantSourceTask.java

```diff
@@ -210,7 +211,7 @@ public void stop() {
             try {
                 Thread.sleep(SHUTDOWN_DELAY_MILLISEC);
             } catch (InterruptedException e) {
-                LOG.error(e);
+                LOG.error(e.getMessage());
```

Comment:
MIght be worth including the throwable so a full stack trace can be logged if desired.
```suggestion
                LOG.error(e.getMessage(), e);
```